### PR TITLE
cursor: pointer;

### LINKED ISSUE #187: Idle > Pointer
Make it so that when the mouse hoves over the "okay" and "leave site" button in the google analytics banner, it turns to a pointer
`cursor: pointer;`

### PR DESCRIPTION
**Pull request by CodeGuy92**
### Resolves:

Fixes #187

### Changes:

Makes all buttons change your cursor to "pointer" icon.

### Local Tests:

No, but nothing can go wrong.

### CODE REVIEW

**Comment by gosoccerboy5** [comment]
ok nice, it seems cool

**Review by BarelySmooth — CHANGES_REQUESTED** [review]
It looks good! But can you place this somewhere above? This is currently below the footer and right now, there are no buttons in the footer.

**Review by Accio1 — CHANGES_REQUESTED** [review]
I agree with @BarelySmooth. Since this applies to all buttons, it should be near the top of the stylesheet.

**Comment by CodeGuy92** [comment]
@Accio1 done

**Comment by Accio1** [comment]
@CodeGuy92 Since the button does not just apply to the navbar, it shouldn't be in with the navbar CSS either. It should be just before the code for the navbar.

File: resources/stylesheet.css [hunk 1]
```diff
@@ -58,6 +58,10 @@ header {
 	font-family: "Montserrat", Arial, sans-serif;
 }
 
+button {
```
Comment (BarelySmooth) [inline resources/stylesheet.css:61]: but this is in the header part. Could you move it a little up?

**Comment by CodeGuy92** [comment]
@Accio1 is this the final commit

**Review by BarelySmooth — APPROVED** [review]
This looks fine! 😃

**Review by gosoccerboy5 — APPROVED** [review]
looks great

**Review by leahcimto — APPROVED** [review]
Looks good!

**Comment by BarelySmooth** [comment]
@CodeGuy92 For some reason the fork you are trying to merge from no longer exists. (That is probably why the test failed).

@Accio1 Is merging this still possible?

**Comment by gosoccerboy5** [comment]
My goodness, can't we have @accio1 just write some CSS and slap it into the file? We've been waiting 10 days, there were a bunch of changes requested, and now the branch is deleted. Why don't we put this behind us and have someone with write access edit the files or at least make a new PR?

**Comment by CodeGuy92** [comment]
> @CodeGuy92 For some reason the fork you are trying to merge from no longer exists. (That is probably why the test failed).
> 
> @Accio1 Is merging this still possible?

What? It is? Where did my fork go?

**Comment by CodeGuy92** [comment]
https://github.com/theforumhelpers/theforumhelpers.github.io/pull/253

**Comment by Accio1** [comment]
> My goodness, can't we have @Accio1 just write some CSS and slap it into the file? We've been waiting 10 days, there were a bunch of changes requested, and now the branch is deleted. Why don't we put this behind us and have someone with write access edit the files or at least make a new PR?

Please appreciate that I have been very busy the past couple weeks working on completing my school year. This was not a priority, so that is why I didn't have time to review it. Please be patient.

**Comment by gosoccerboy5** [comment]
> > My goodness, can't we have @Accio1 just write some CSS and slap it into the file? We've been waiting 10 days, there were a bunch of changes requested, and now the branch is deleted. Why don't we put this behind us and have someone with write access edit the files or at least make a new PR?
> 
> @gosoccerboy5 Please appreciate that I have been very busy the past couple weeks working on completing my school year. This was not a priority, so that is why I didn't have time to review it. Please be patient.

I apologize for being irrational like that..

**Pull request closed without merging**